test/system/ssh_keys_test.go

```diff
@@ -78,12 +78,16 @@ spec:
       terminationGracePeriodSeconds: 1
       containers:
       - name: test-ssh-keys
-        image: ubuntu
-        command: ["bash"]
+        image: quay.io/bitnami/nginx
+        command: ["/bin/bash"]
         volumeMounts:
         - name: ssh
           mountPath: /home/core/.ssh
           readOnly: true
+				securityContext:
+          runAsNonRoot: true
+          runAsUser: 65534
+          runAsGroup: 65534
```

Comment:
Indentation + I think we should be running as root here :wink: Or as core user UID on the host.